projects/vdk-plugins/vdk-oracle/src/vdk/plugin/oracle/ingest_to_oracle.py

```diff
@@ -18,12 +20,90 @@
 log = logging.getLogger(__name__)
 
 
+# Functions for escaping special characters
+def _is_plain_identifier(identifier: str) -> bool:
+    class InvalidOracleIdentifierError(BaseException):
```

Comment:
The custom InvalidOracleIdentifierError is great. Please move it outside the _is_plain_identifier function. It’s more conventional to define custom exceptions at the module level rather than inside a function otherwise importing and catching is more complex.